### PR TITLE
open-iscsi: add runit service

### LINKED ISSUE #40861: Add a service for iscsid (open-iscsi)
When the `open-iscsi` package is installed on Arch Linux, a systemd service is created which essentially calls `/usr/bin/iscsid -f` when started:
```shell
$ cat /etc/systemd/system/multi-user.target.wants/iscsid.service
[Unit]
Description=Open-iSCSI
Documentation=man:iscsid(8) man:iscsiuio(8) man:iscsiadm(8)
DefaultDependencies=no
After=network-online.target iscsiuio.service iscsi-init.service
Before=remote-fs-pre.target
Wants=remote-fs-pre.target
Requires=iscsi-init.service

[Service]
Type=notify
NotifyAccess=main
ExecStart=/usr/bin/iscsid -f  <-----
KillMode=mixed
Restart=on-failure

[Install]
WantedBy=multi-user.target
Also=iscsid.socket
```

I wanted to run Void as a server for testing purposes and I need `iscsid` to run on there. I've created the following service file as a first:
```shell
$ cat /etc/sv/iscsid/run 
#!/bin/sh
[ -r conf ] && . ./conf
exec iscsid -f

$ sv status /var/service/iscsid
run: /var/service/iscsid: (pid 825) 311s
```

`iscsiuio` doesn't start for me for some reason, but `iscsid` is happy as far as I can tell. I'm quite sure that `iscsi-init.service` is NOT needed, since that only creates a `/etc/iscsi/initiatorname.iscsi` file which the XBPS package installation already does AFAIK.

My question/request: Do we need that service? If so, is my `/etc/sv/iscsid/run` enough? :)
Thank you!

### PR DESCRIPTION
**Pull request by thetredev**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, `x86_64-glibc`

#### Notes
- This fixes #40861

### CODE REVIEW

**Comment by classabbyamp** [comment]
yes, revbump, and don't skip ci

File: srcpkgs/open-iscsi/files/iscsid/run [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+exec 2>&1
+[ -r conf ] && . ./conf
```
Comment (paper42) [inline srcpkgs/open-iscsi/files/iscsid/run:3]: what's the point of this source when we are not using any variables from it?

Comment (thetredev) [reply srcpkgs/open-iscsi/files/iscsid/run:3]: Good point, I'll remove it.

**Comment by thetredev** [comment]
@mhmdanas @classabbyamp I'll remove the skip CI tag. But I don't know what you mean by revbump. Could you please explain?

**Comment by classabbyamp** [comment]
increment the value of `revision` so the package gets rebuilt and people get the updated version

**Comment by thetredev** [comment]
Should be OK now.

File: srcpkgs/open-iscsi/files/iscsid/run (hunk 1 above)
Comment (thetredev) [reply srcpkgs/open-iscsi/files/iscsid/run:3]: done.

Comment (mhmdanas) [reply srcpkgs/open-iscsi/files/iscsid/run:3]: Did you forget to commit or push your removal of this line? It's still there.

Comment (thetredev) [reply srcpkgs/open-iscsi/files/iscsid/run]: oops, forgot to add, re-check now please.

**Comment by thetredev** [comment]
I think the commit message and PR title should be changed to **open-iscsi: Add iscsid runit service**, because I'm not sure if we need the `iscsiuio` service later (as discussed in the issue linked in the PR description). What do you think?

**Review by vincele — APPROVED** [review]
LGTM

**Comment by classabbyamp** [comment]
rebase on master please

**Comment by thetredev** [comment]
fixed merge conflict

**Comment by classabbyamp** [comment]
cc @leahneukirchen

**Pull request merged**